src/main/java/tahub/contacts/model/course/Attendance.java

```diff
@@ -0,0 +1,85 @@
+package tahub.contacts.model.course;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the attendance of a student in the address book.
+ */
+public class Attendance {
+    private final ArrayList<Boolean> attendanceList;
+
+    /**
+     * Constructs an {@code Attendance} object.
+     */
+    public Attendance() {
+        attendanceList = new ArrayList<>();
+    }
+
+    /**
+     * Constructs an {@code Attendance} object from an attendance list.
+     *
+     * @param attendanceList Preexisting attendance list as a {@link List} of {@link Boolean}.
+     */
+    public Attendance(List<Boolean> attendanceList) {
+        this.attendanceList = new ArrayList<>(attendanceList);
+    }
+
+    /**
+     * Adds a new lesson marked as attended.
+     */
+    public void addAttended() {
+        attendanceList.add(true);
+    }
+
+    /**
+     * Adds a new lesson marked as absent.
+     */
+    public void addAbsent() {
+        attendanceList.add(false);
+    }
```

Comment:
`addAttended` and `addAbsent`.  Maybe can make the method names clearer

Reply:
Ok